CefSharp.Core/Internals/ClientAdapter.cpp

```diff
@@ -737,6 +738,40 @@ namespace CefSharp
                 StringUtils::ToClr(scheme), callbackWrapper);
         }
 
+		bool ClientAdapter::OnSelectClientCertificate(CefRefPtr<CefBrowser> browser, bool isProxy, const CefString& host,
+			int port, const CefRequestHandler::X509CertificateList& certificates, CefRefPtr<CefSelectClientCertificateCallback> callback) {
+			
+			auto handler = _browserControl->RequestHandler;
+
+			if (handler == nullptr)
+			{
+				return false;
+			}
+
+			auto browserWrapper = GetBrowserWrapper(browser->GetIdentifier(), browser->IsPopup());
+			auto callbackWrapper = gcnew CefCertificateCallbackWrapper(callback, certificates);
+
+			auto list = gcnew System::Security::Cryptography::X509Certificates::X509Certificate2Collection();
```

Comment:
Add a `using statement` for `System::Security::Cryptography::X509Certificates`, just to tidy these up a bit.